src/OnboardingSPA/components/LivePreview/BlockPreview/index.js

```diff
@@ -76,10 +76,9 @@ const BlockPreview = ( {
 		}
 	}, [ storedPreviewSettings ] );
 
-	return (
-		<div className={ `live-preview__container-${ styling }` }>
-			{ loading && (
-				<div className="live-preview__container--is-skeleton">
+	const SkeletonLivePreview = memo(() => {
+		return (
+			<div className="live-preview__container--is-skeleton">
```

Comment:
Hey
Everything looks great, let's just resolve the conflict here, TIA.

Reply:
Isn't this fixed already?